cxx/gendb.cc

```diff
@@ -304,7 +356,7 @@ double GenDB::unincorporate_reference_relation(
              std::unordered_map<T_items, ObservationVariant, H_items>>&
         stored_value_map) {
   // We can stop unincorporating from base relations if the index of
-  // the domain of interest is greater than the number of domains.
+  // the domain of interest is greater than the max index.
```

Comment:
max index of what?

Reply:
I meant the max index of the vector of items, so the length minus one. I rephrased the comment.